pkg/resolution/resolver/cluster/resolver.go

```diff
@@ -183,7 +197,16 @@ func (r *ResolvedClusterResource) Annotations() map[string]string {
 // Source is the source reference of the remote data that records where the remote
 // file came from including the url, digest and the entrypoint.
 func (r ResolvedClusterResource) Source() *pipelinev1beta1.ConfigSource {
-	return nil
+	h := sha256.New()
+	h.Write(r.Content)
+	sha256CheckSum := hex.EncodeToString(h.Sum(nil))
```

Comment:
I'm curious why using hex not base64?

Reply:
Thanks for asking.
Just trying to follow how to calculate for hub resource, and this [discussion](https://github.com/tektoncd/pipeline/pull/5666#discussion_r1002301051) might be related.
I think those are functional equivalent, and hex approach is mainly for displaying purpose as called out [here](https://stackoverflow.com/questions/10701874/generating-the-sha-hash-of-a-string-using-golang/10701951#10701951).

Reply:
I mean base64 encoding, hex encoding is actually base16. And it seems base64 is more efficient?
https://stackoverflow.com/questions/3183841/base64-vs-hex-for-sending-binary-content-over-the-internet-in-xml-doc#:~:text=The%20difference%20between%20Base64%20and,it's%20more%20efficient%20than%20hex.

Reply:
Oh I'm not sure which one we should use here. Or what is the standard way of doing this. Just want to discuss this

Reply:
I think it just doesn't matter that much as long as we can compare if anything is tempered by checking digest. Given base64 can help save space, I think it's fine to use.

Reply:
Just realized that intoto recommended the [hex-encoded value](https://github.com/in-toto/attestation/blob/main/spec/field_types.md#DigestSet:~:text=map%20from%20algorithm%20name%20to%20lowercase%20hex%2Dencoded%20value) in the DigestSet. So we might lean on hex for now. Thanks for sharing hex v.s. base64 though. That's good to know!